.github/workflows/syncTestTestnet.yml

```
@@ -48,17 +48,26 @@
         env:
-          scheme: ${{ 'default' }}
+          scheme: ${{ 'DashSync-Example' }}
           platform: ${{ 'iOS Simulator' }}
         run: |
-          xcodebuild test-without-building -scheme "DashSync-Example" -workspace "DashSync.xcworkspace"  -destination "platform=$platform,name=iPhone 13" -testPlan TestnetSyncTests
+          xcodebuild test-without-building \
+            -scheme "DashSync-Example" \
+            -workspace "DashSync.xcworkspace" \
+            -destination "platform=$platform,name=iPhone 13" \
+            -testPlan TestnetSyncTests \
+            CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGN_ENTITLEMENTS="" CODE_SIGNING_ALLOWED=NO
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)